sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -9,6 +9,76 @@ import { AbortSignalLike } from '@azure/abort-controller';
 // @public
 export type CancelOnProgress = () => void;
 
+// @public
+export function createGetLROStatusFromResponse<TResult>(lroPrimitives: LRO<TResult>, config: LROConfig, finalStateVia?: FinalStateVia): (rawResponse: RawResponse, flatResponse: TResult) => LROStatus<TResult>;
+
+// @public
+export type FinalStateVia = "azure-async-operation" | "location" | "original-uri";
+
+// @public
+export type GetLROStatusFromResponse<T> = (rawResponse: RawResponse, flatResponse: T) => LROStatus<T>;
+
+// @public
+export interface LRO<T> {
+    requestMethod: string;
+    requestPath: string;
+    retrieveAzureAsyncResource: (path?: string) => Promise<LROStatus<T>>;
+    sendInitialRequest: (initializeState: (rawResponse: RawResponse, flatResponse: unknown) => boolean) => Promise<LROResponse<T>>;
+    sendPollRequest: (config: LROConfig, path: string) => Promise<LROStatus<T>>;
+}
+
+// @public
+export interface LROBody extends Record<string, unknown> {
+    properties?: {
+        provisioningState?: string;
```

Comment:
I assume there is some service-specific reason why provisioningState is in two places on this type?

Reply:
Yeah I think it is just a case of many LRO conventions out there. However now that I am thinking about it, we could probably hide this and cast to it internally.

Reply:
I hid `LROBody`.